libcloudforensics/gcp.py

```diff
@@ -718,38 +717,38 @@ def add_labels(self, new_labels_dict, blocking_call=False):
     """
 
     get_operation = self.get_operation().execute()
-    labelFingerprint = get_operation['labelFingerprint']
+    label_fingerprint = get_operation['label_fingerprint']
 
-    exisitng_labels_dict = dict()
+    existing_labels_dict = dict()
     if self.get_labels() is not None:
-      exisitng_labels_dict = self.get_labels()
-    exisitng_labels_dict.update(new_labels_dict)
-    labels_dict = exisitng_labels_dict
-    request_body = {'labels': labels_dict, 'labelFingerprint': labelFingerprint}
+      existing_labels_dict = self.get_labels()
+    existing_labels_dict.update(new_labels_dict)
+    labels_dict = existing_labels_dict
+    request_body = {'labels': labels_dict, 'label_fingerprint': label_fingerprint}
 
     resource_type = self.get_resource_type()
     operation = None
     if resource_type not in ['compute#instance', 'compute#snapshot',
                              'compute#disk']:
       raise RuntimeError((
-          'Compute resource Type {0:s} is not one of the defined types in '
-          'libcloudforensics library (Instance, Disk or Snapshot) '
-      ).format(resource_type))
+                           'Compute resource Type {0:s} is not one of the defined types in '
+                           'libcloudforensics library (Instance, Disk or Snapshot) '
+                         ).format(resource_type))
```

Comment:
Consider building this string in a `msg =` variable before for better readability

Reply:
Addressed.